Content.Shared/Devour/Components/DevourerComponent.cs

```diff
@@ -71,6 +72,18 @@ public sealed partial class DevourerComponent : Component
     [ViewVariables(VVAccess.ReadWrite), DataField("healRate")]
     public float HealRate = 15f;
 
+    /// <summary>
+    /// The chemical ID's injected into the victim upon being devoured.
+    /// </summary>
+    [DataField("stomachChemicals", customTypeSerializer: typeof(PrototypeIdListSerializer<ReagentPrototype>))]
+    public List<string> StomachChemicals = new();
+
+    /// <summary>
+    /// The amount of stomach chemicals  injected into the devoured entity.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("damageRate")]
+    public float DamageRate = 50f;
```

Comment:
DamageRate does not correspond to what this is actually doing as it has nothing to do with damage.

You're better off just using a solution instead.

Reply:
I tried following the naming already used in the component, the datafield keeping track of the amount of ichor injected was called HealRate.